command/genesis/params.go

```diff
@@ -110,6 +122,12 @@ func (p *genesisParams) validateFlags() error {
 		return errValidatorsNotSpecified
 	}
 
+	if p.isPolyBFTConsensus() {
+		if err := p.extractTokenParams(); err != nil {
+			return errInvalidTokenParams
```

Comment:
Just propagate the error?
```suggestion
			return err
```

Reply:
Fixed: https://github.com/0xPolygon/polygon-edge/pull/1376/commits/ab7a094805b4e5da93a395b0dd84f32cf7f251e8